app/src/main/res/drawable/ic_send.xml

```diff
@@ -4,6 +4,6 @@
         android:viewportWidth="24.0"
         android:viewportHeight="24.0">
     <path
-        android:fillColor="#dd000000"
+        android:fillColor="@color/colorAccent"
```

Comment:
Older versions of Android have a problem with using color references in drawables.